modin/data_management/utils.py

```diff
@@ -31,17 +39,25 @@ def get_default_chunksize(length, num_splits):
 
 
 def compute_chunksize(df, num_splits, default_block_size=32, axis=None):
-    """Computes the number of rows and/or columns to include in each partition.
+    """
+    Compute the number of rows and/or columns to include in each partition.
 
-    Args:
-        df: The DataFrame to split.
-        num_splits: The maximum number of splits to separate the DataFrame into.
-        default_block_size: Minimum number of rows/columns (default set to 32x32).
-        axis: The axis to split. (0: Index, 1: Columns, None: Both)
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        DataFrame to split.
+    num_splits : int
+        Number of splits to separate the DataFrame into.
+    default_block_size : int, default: 32
+        Minimum number of rows/columns in a single split.
+    axis: {0: Index, 1: Columns, None: Both}
```

Comment:
should be either `int, optional` or `{None, 0 1}` with explanation of values in the description